### ISSUE TITLE
gtx 1070 "driver stopped responding and has recovered " video card not found in the system

### ISSUE THREAD

**Issue by timothyco89** [nicehash/NiceHashMiner-Archived#1179]
gtx 1070 video card
when mining sometimes it will stop and pop out an error driver
 "stopped responding and has recovered"
and the video card will not be visible in the system and i have to restart my computer
pls help

**Comment by Stephen-Randall** [comment]
@timothyco89 Do you have a single card? Are you overclocking? Are you using a PCI-E Riser?

Most of the Driver Stopped Responding messages are due to pushing Core or Mem clocks to far out of specification. 

I have found that occasionally my card will drop and the driver will freeze due to a flaky PCI Riser.

I do not suspect that Nicehash Miner is causing this issue.

**Comment by dogcatchescar** [comment]
Are you using Multiple GPUs? Riser Cards?

The Display Driver has stopped responding message is usually caused by overclocking. If your Core or Mem is too high consider dropping it back a bit.

I notice that occasionally  my PCIE riser will drop the card completely (No longer shows in Device Manager) and the error message will pop up. 

I do not suspect this issue is caused by NiceHash Miner.

**Comment by OneOfMany07** [comment]
Yeah, I was guessing either overheating eventually causes an error, or not enough power (if supply is underrated for the load).

**Comment by timothyco89** [comment]
single gpu 
no riser card
no overclock 
750w power supply seasonic 80gold

**Comment by Squall-Leonhart** [comment]
just toggle the card in device manager.
